_artifacts/opensearch-1.3/arm64/opensearch-1.3.2-linux-arm64-yum.markdown

```diff
@@ -0,0 +1,13 @@
+---
+role: daemon
+artifact_id: opensearch
+version: 2.0.0-rc1
+platform: linux
+architecture: arm64
+slug: opensearch-2.0.0-rc1-linux-arm64-yum
+category: opensearch
+type: yum
+guide: https://opensearch.org/docs/latest/opensearch/install/rpm
+artifact_url: https://artifacts.opensearch.org/releases/bundle/opensearch/2.x/opensearch-2.x.repo
```

Comment:
The version and URL for this file will need to be changed.

Reply:
fixed